ibis/backends/pyspark/tests/test_ddl.py

```diff
@@ -194,19 +172,6 @@ def table(con, temp_database):
     con.drop_table(table_name, database=temp_database)
 
 
-def test_change_properties(con, table, temp_database):
```

Comment:
Any particular reason to get rid of this test?

Reply:
Because I removed the `PySparkTable` class meaning there is no `table.alter()` method anymore. Since these are eager ddl operations they should be implemented on the backend.